### PR TITLE
Added reflective IR event and also second getter with all values

### PR DESCRIPTION
**Pull request by shamlian**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3298,6 +3549,180 @@ Touch Sensor changed event. The robot sends a Touch Sensor Event whenever one or
 
 Note that the Create 3 robot maps button 1 (·) to `FL` and button 2 (··) to `FR`.
 
+### Device 19 - Docking Sensors
```
Comment (raphaelcherney) [inline README.md]: @shamlian FYI, this PR also includes the Docking Sensors from #18 (probably unintentional?)

Comment (shamlian) [reply README.md]: I don't know what you're talking about 😇

**Comment by shamlian** [comment]
Need to add getter for thresholds

**Comment by shamlian** [comment]
decision to harmonize responses for cmd 0 and 2, and pack sensors into 12-bit values so they all fit

**Review by raphaelcherney — APPROVED** [review]

**Pull request merged**